test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -205,24 +205,19 @@ public async Task Scenario_Logging()
 
             // verify log levels in traces
             LogMessage[] traces = Fixture.Host.GetScriptHostLogMessages(userCategory).Where(t => t.FormattedMessage != null && t.FormattedMessage.Contains("loglevel")).ToArray();
-            Assert.Equal(LogLevel.Information, traces[0].Level);
-            Assert.Equal("loglevel default", traces[0].FormattedMessage);
-            Assert.Equal(LogLevel.Information, traces[1].Level);
-            Assert.Equal("loglevel info", traces[1].FormattedMessage);
-            Assert.Equal(LogLevel.Trace, traces[2].Level);
-            Assert.Equal("loglevel verbose", traces[2].FormattedMessage);
-            Assert.Equal(LogLevel.Warning, traces[3].Level);
-            Assert.Equal("loglevel warn", traces[3].FormattedMessage);
-            Assert.Equal(LogLevel.Error, traces[4].Level);
-            Assert.Equal("loglevel error", traces[4].FormattedMessage);
+            Assert.True(traces.Any(t => t.Level == LogLevel.Information && t.FormattedMessage == "loglevel default" ));
```

Comment:
This Node.js test is historically pretty flaky and was failing for my PR, so I went ahead and improved it. We shouldn't test the exact order of messages because the order isn't guaranteed across the RPC channel.

Btw, here's proof the test failed even before my PR: https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=147520&view=ms.vss-test-web.build-test-results-tab&runId=1345189&resultId=100017

Reply:
I have also been victimized by this flaky test :)

cc: @brettsam because he has also looked into this before

Reply:
This is great! I think we can mark #9439 as fixed? @satvu you didn't mention the specific test that was failing in that issue, but this must have been it, right?

And this actually is related to the "out-of-order processing" that I'm looking into here -- https://github.com/Azure/azure-functions-host/issues/9238. My guess is that this test ran fine for years, then after the re-structuring of the worker pipeline (which introduced this ordering problem), it's become flaky for the last ~9 months or so.

Reply:
I didn't realize the out-of-order processing happened that recently. It's a relatively minor problem, but it annoys people often enough that I'd still love to see it fixed.

I closed #9439 with this PR, but feel free to ping me if you're seeing more Node.js flaky test failures and I'd be happy to help investigate.